be/src/exec/hdfs_scanner.cpp

```diff
@@ -468,6 +468,9 @@ Status HdfsScannerContext::append_or_update_not_existed_columns_to_chunk(ChunkPt
         ck->append_or_update_column(std::move(col), slot_desc->id());
     } else {
         for (auto* slot_desc : not_existed_slots) {
+            if (ck->is_slot_exist(slot_desc->id())) {
+                continue;
+            }
```

Comment:
Does it hurt performance if there are thousands of columns in the chunk?

Reply:
I think it is ok now, thousands of columns in the chunk is rare and, the column num append to chunk is small, the data structure in chunk for detect id whether exist is  map